src/Security.tsx

```diff
@@ -23,9 +22,8 @@ const Security: React.FC<{
 } & React.HTMLAttributes<HTMLDivElement>> = ({ 
   oktaAuth, 
   onAuthRequired, 
-  children 
```

Comment:
accept `restoreOriginalUri ` here to avoid reloading the page during after login redirect.